projects/qcms/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: "https://hg.mozilla.org/mozilla-central/file/tip/gfx/qcms/"
+primary_contact: "agaynor@mozilla.com"
```

Comment:
We should probably also CC someone either from the fuzzing team or the gfx team -- @choller any iea who that should be?

Reply:
Maybe @posidron would be the right person since he did qcms fuzzing before from what I remember.